cargo-pgrx/README.md

```diff
@@ -509,6 +514,8 @@ Options:
   -r, --release                        compile for release mode (default is debug)
       --profile <PROFILE>              Specific profile to use (conflicts with `--release`)
   -n, --no-schema                      Don't regenerate the schema
+      --runas <RUNAS>                  Use `sudo` to initialize and run the Postgres test instance as this system user
```

Comment:
This should look more like
```suggestion
      --runas <USER>                  Use `sudo` to initialize and run the Postgres test instance as this system user
```

Reply:
this is what clap generates for us.  I don't see how to change that.